doc/MAGIC.md

```diff
@@ -178,13 +178,15 @@ Below is a table of currently implemented effects, along with special rules for
 | `mod_moves` | Adds damage() moves to the targets.  Negative values "freeze" for that amount of time.
 | `morale` | Gives a morale effect to NPCs or the avatar within the aoe.  Uses damage() for the value.  `decay_start` is duration() / 10.
 | `mutate` | Mutates the targets.  If `effect_str` is defined, mutates toward that category instead of picking at random.  If the `MUTATE_TRAIT` flag is used, allows `effect_str` to be a specific trait.  Damage() / 100 is the percent chance the mutation will be successful (10000 represents 100.00%).
+| `noise` | Causes damage() amount of noise at the target.  Note: the noise can be described further with `sound_type`, `sound_description`, `sound_ambient`, `sound_id` and `sound_variant`.
 | `pain_split` | Evens out all of your limbs' damage.
 | `pull_target` | Attempts to pull the target towards the caster in a straight line.  If the path is blocked by impassable furniture or terrain, the effect fails.
 | `recover_energy` | Recovers an energy source equal to damage of the spell.  The energy source is defined in `effect_str` and may be one of `BIONIC`, `FATIGUE`, `PAIN`, `MANA` or `STAMINA`.
 | `remove_effect` | Removes `effect_str` effects from all creatures in the aoe.
 | `remove_field` | Removes a `effect_str` field in the aoe.  Causes teleglow of varying intensity and potentially teleportation depending on field density, if the field removed is `fd_fatigue`.
 | `revive` | Revives a monster like a zombie necromancer.  The monster must have the `REVIVES` flag.
 | `short_range_teleport` | Teleports the player randomly range spaces with aoe variation.  See also the `TARGET_TELEPORT` and `UNSAFE_TELEPORT` flags.
+| `slime_split` | The slime splits into two large or normal slimes, depending on mass.  Note: seems hardcoded for `mon_blob`-type enemies, check the monster `death_function` + spell `summon` combination.
```

Comment:
```suggestion
| `slime_split` | The slime splits into two large or normal slimes, depending on mass.  Note: hardcoded for `mon_blob`-type enemies, check the monster `death_function` + spell `summon` combination.
```
This is supposed to be the technical documentation from people making the game, it's supposed to tell how things are actually implemented not make guesses at how it might be ^^"

Reply:
My bad, accepting the suggestion.